pytest_watch/watcher.py

```diff
@@ -164,8 +173,11 @@ def watch(directories=[], ignore=[], auto_clear=False, beep_on_failure=True,
         observer.join()
     except KeyboardInterrupt:
         observer.stop()
-    if onexit:
-        os.system(onexit)
+        if onsigint:
+            os.system(onsigint)
+    else:
+        if onexit:
+            os.system(onexit)
```

Comment:
Poking around with this today, I noticed with the `else`, `onexit` will never get called (unless there was a problem with ptw). I think `onexit` should always be called regardless, since that's its intention.

Why couldn't you have used `onexit` before?